tests/by-util/test_ls.rs

```diff
@@ -4298,6 +4297,100 @@ fn test_ls_context_format() {
     }
 }
 
+/// Helper function to validate `SELinux` context format
+/// Returns the context parts if valid, panics with descriptive message if invalid
+#[cfg(feature = "feat_selinux")]
+fn validate_selinux_context(context: &str) -> Vec<&str> {
+    assert!(
+        context.contains(':'),
+        "Expected SELinux context format (user:role:type:level), got: {}",
+        context
+    );
```

Comment:
This `assert` is unnecessary because the `assert_eq` on lines 4311 - 4316 will also fail if there is no colon.

Reply:
yeah, forgot to call the function here :)